py3status/modules/mysql_slave.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+"""
+Display MySQL/MariaDB slave state.
+
+This module displays the number of seconds the slave is behind the master.
+
+Configuration parameters:
+    cache_timeout: refresh cache_timeout for this module (default 5)
+    format: display format for this module
+        *(default '\?color=seconds_behind_master {host} is '
+        '[\?if=seconds_behind_master {seconds_behind_master}s behind|master]') *
+    host: the host to connect to (default None)
+    passwd: the password to use for the connection (default None)
+    port: the port to connect to (default 3306)
```

Comment:
We can kill those configs with a dict `parameters` by replacing `conn = connect(user=self.user, passwd=self.passwd, host=self.host, port=self.port)` with `x = connect(**self.parameters).cursor().execute(self.query)` to support all available `connect()` parameters.